example/glue/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -197,16 +213,12 @@ MM_CollectorLanguageInterfaceImpl::parallelDispatcher_handleMasterThread(OMR_VMT
 void
 MM_CollectorLanguageInterfaceImpl::generationalWriteBarrierStore(OMR_VMThread *omrThread, omrobjectptr_t parentObject, fomrobject_t *parentSlot, omrobjectptr_t childObject)
 {
```

Comment:
Will add concurrent/scavenger barriers inline in writeAccessBarrierStore(); unfortunately not possible to inline entire method in CollectorLanguageInterface.hpp as there are circular includes dependencies in that case.